### ISSUE TITLE
Make running tests easy, e.g. by tox

### ISSUE THREAD

**Issue by vlcinsky** [crsmithdev/arrow#117]
Attempting to create Czech locale, I was happy to find test suite in place.

However, running the test suite is difficult (and I did not get through).
# Reasons making running tests difficult
## using `make`

Testing seems to be run by

```
$ make test
```

However, `make is not always available.
## Unspoken assumptions

The test in `Makefile` assumes, there is virtualenv in `local/bin` subdirectory.
## Difficulties to import `chai`

```
$ nosetest tests
```

is failing with errors like:

```
======================================================================
ERROR: Failure: ImportError (No module named chai)
----------------------------------------------------------------------
Traceback (most recent call last):
  File "/usr/local/lib/python2.7/dist-packages/nose/loader.py", line 413, in loadTestsFromName
    addr.filename, addr.module)
  File "/usr/local/lib/python2.7/dist-packages/nose/importer.py", line 47, in importFromPath
    return self.importFromDir(dir_path, fqname)
  File "/usr/local/lib/python2.7/dist-packages/nose/importer.py", line 94, in importFromDir
    mod = load_module(part_fqname, fh, filename, desc)
  File "/home/javl/sandbox/arrow/arrow/tests/util_tests.py", line 3, in <module>
    from chai import Chai
ImportError: No module named chai

----------------------------------------------------------------------
Ran 7 tests in 0.002s
```

Even manually installing `chai` did not help, the error above appears even when `chai` is installed:

```
$ pip freeze|grep chai
chai==0.4.8
```
# Proposal: use `tox`
## `tox` is written in Pyhton

No need to have `make` installed.
## Allows using `nose`

`tox` is not forcing any testing tool, using command `nosetests` is natural.
## Installs test dependencies

`tox` allows specifying, what modules are required for testing. Here it would be `chai`, `nose` and
may be few more.
## Manges testing under various python versions

`tox` makes running tests under different python versions easy. All the tests are performed in virtualenv, no need to care about setting it up.

In case such a change sounds acceptable, I would try to modify existing framework for this, trying to preserve as much of existing test suite unchanged.

**Comment by bukzor** [comment]
This is fixed in #347

**Comment by andrewelkins** [comment]
Confirmed is fixed in 0.9.0
